dll/shellext/zipfldr/zlib_contrib/MiniZip64_info.txt

```diff
@@ -0,0 +1,74 @@
+MiniZip - Copyright (c) 1998-2010 - by Gilles Vollant - version 1.1 64 bits from Mathias Svensson
```

Comment:
I think zlib_contrib should be moved to zlib.

Reply:
You mean, inside our existing zlib?

Reply:
Being close to linux may not be bad. Some it is more easy if it's an bit more equivalent in direction linux as the "original", because later it's more easy to use with.. :)

Reply:
Since they come from the same source, keeping them together makes it easier to sync.

Reply:
Mark, do not mob around here otherwise I'll report you to Github.

Reply:
@blackcrack : I don't understand your first message either...

Reply:
ah this is a other musik :) Okey maybe a bit Awkwardly expressed :)
i have meant , because Linux use the zlib also because i have mentored to use the zlib too.
Makes may later usable if we use it directly.. {less complications or so.. (it was just an assumption)} ;)